cloudant-client/src/main/java/com/cloudant/client/api/model/FindByIndexOptions.java

```diff
@@ -120,10 +121,10 @@ public FindByIndexOptions useIndex(String designDocument) {
      * @return this to set additional options
      */
     public FindByIndexOptions useIndex(String designDocument, String indexName) {
-        JsonPrimitive jsonDesign = new JsonPrimitive(designDocument);
-        JsonPrimitive jsonIndex = new JsonPrimitive(indexName);
-        this.useIndex.add(jsonDesign);
-        this.useIndex.add(jsonIndex);
+        JsonArray index = new JsonArray();
+        index.add(new JsonPrimitive(designDocument));
+        index.add(new JsonPrimitive(indexName));
+        this.useIndex = index;

```

Comment:
Do either of the `useIndex` methods need to check the args to the method are not null?